src/main/flight/pid.c

```diff
@@ -291,6 +291,20 @@ void pidResetIterm(void)
     }
 }
 
+void pidUpdateTpaFactor(float throttle)
+{
+    pidRuntime.tpaFactor = 1.0f; 
+    const float tpaCut = currentControlRateProfile->dynThrPID / 100.0f;
+    const float tpaBreakpoint = (currentControlRateProfile->tpa_breakpoint - 1000) / 1000.0f;
+    if (throttle > tpaBreakpoint) {
```

Comment:
We can move line 297
```
const float tpaCut = currentControlRateProfile->dynThrPID / 100.0f;
```
into the if-statement right below this line (line 299). So `tpaCut` will only be calculated if necessary (if throttle > tpaBreakpoint).

Reply:
Thanks, will do